_posts/2018-06-04-baring-it-all-in-your-whitepaper.md

```diff
@@ -0,0 +1,41 @@
+
+If you meet resistance to this approach, it should be easy to justify why each of these approaches is desirable. There's no point in committing to features for a product that is still very conceptual, if people invest because they are expecting a specific feature you are likely going to end up disappointing more than one of your investors. They will all see different features in their head when they invest initially because you haven't been specific about them. They key is to commit to when they will have clarity around their belief in your vision and execution capacity. On the other hand there's no point in committing to aggressive timelines if you have committed to specific features. If people have invested in the amazing features you have clearly defined, then they should understand that perfecting them takes time. Even when they are clearly defined you still need to finesse and test with your market, this all takes time.
+
+Two approaches that are open and provide clarity without digging yourself a hole to dragged into later on in the process.
+
+## About giving up your seekrits!
+
+In terms of product features etc - I tend to take a slightly different view to many others, so caveat emptor on the following ramblings from me.
+
+I don't believe in competitors, there may be other companies that validate my target market, or create disenfranchised customers for me to pick up. They are both great things. Ultimately success that I've seen has been more based on the support of those who believe and connect with the world view of your product. That comes back to some very lean principles; get out of the building, connect, learn, fail, then produce something that resonates with your customers. There's always going to be people creating similar products.
+
+I see it like offices, you walk into a business and make a judgement about how you feel about them based on the way they decorate their office. Some you will connect with, others less so. We can't build the product that satisfies everyone, so make your "office" awesome for you and those close to you, that passion will translate into awesome relationships with your customers and a product experience they connect with.
+
+Now the more blockchain related perspective: TSEs that are most successful are the ones that understand that they are creating something that will ultimately be controlled by the community. The products that fit into this view need to be understood by the community because protocols are only useful if anyone can implement them. If you're running a TSE for a product that will never play nice the community then you're most likely either: a) not being ambitious enough or b) probably not an ideal candidate for running a TSE. Hopefully you're just in (a) because that just means thinking bigger!
```

Comment:
"Token sale events (TSEs)"

First time use in the post, most people don't know what "TSE" stands for

Reply:
We're avoiding ICO as it carries legal grey area BS about securities...